src/main/java/boot/config/SwggerConfig.java

```diff
@@ -0,0 +1,19 @@
+package boot.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwggerConfig implements WebMvcConfigurer {
+
```

Comment:
What about @Bean Docket? Your swagger work?

Reply:
Yes it is working